lib/compress/inline.js

```diff
@@ -148,18 +150,58 @@ function scope_encloses_variables_in_this_scope(scope, pulled_scope) {
     return false;
 }
 
+/**
+ * A extra check function for `top_retain` option, compare the length of const identifier
+ * and init value length and return true if init value is longer than identifier. for example:
+ * ```js
+ * // top_retain: ["example"]
+ * const example = 100
+ * ```
+ * it will return false because length of "100" is short than identifier "example".
+ * 
+ * ### Why Need this function
+ * see issue 1413. Put it simple, if identifier in top_retain option, terser will retain it,
+ * but when identifier be referenceed, we might be can inline the identifier if length of 
+ * value is shorter than identifier. so we add this function to compare the length of init
+ * value and identifier.
+ * 
+ * @param {AST_Node} def definition of variable
+ * @param {AST_Node} fixed_value init value of const, maybe null if def is not const
+ * @return {bool}
+ */
```

Comment:
```suggestion
/**
 * An extra check function for `top_retain` option, compare the length of const identifier
 * and init value length and return true if init value is longer than identifier. for example:
 * ```
 * // top_retain: ["example"]
 * const example = 100
 * ```
 * it will return false because length of "100" is short than identifier "example".
 */
```